src/back/project/src/model/cc_type_param_serialize.cpp

```diff
@@ -0,0 +1,31 @@
+#include "cc_type_param_serialize.hpp"
+
+#include <userver/formats/json/value_builder.hpp>
+
+namespace svetit::project::model {
+
+formats::json::Value Serialize(
+	const CcTypeParam& ccTypeParam,
+	formats::serialize::To<formats::json::Value>)
+{
+	formats::json::ValueBuilder builder{};
+
+	builder["ccTypeId"] = ccTypeParam.ccTypeId;
+	builder["paramId"] = ccTypeParam.paramId;
+	builder["isDeleted"] = ccTypeParam.isDeleted;
```

Comment:
Давай мапить isDeleted, только если он true
```suggestion
	if (ccTypeParam.isDeleted)
		builder["isDeleted"] = ccTypeParam.isDeleted;
```

Reply:
Готово.